### PR TITLE
Add a User-Agent when using the Lambda Runtime API

### PR DESCRIPTION
**Pull request by bmoffatt**
Addresses #96 - This will help you stand out in a crowd of "Unknown/Unknown" in our logs

### CODE REVIEW

File: Sources/AWSLambdaRuntimeCore/HTTPClient.swift [hunk 1]
```diff
@@ -26,6 +26,7 @@ internal final class HTTPClient {
 
     private var state = State.disconnected
     private var executing = false
+    private let headers = HTTPHeaders([("User-Agent", "Swift-Lambda/Unknown")])
```
Comment (weissi) [inline Sources/AWSLambdaRuntimeCore/HTTPClient.swift:29]: I'd do this one `static` so we can save a few allocations (and a little bit of space). Given that we're editing this line anyway, I'd also make the `user-agent` lower-case (just to be more compatible with HTTP/2 if Lambda were to ever switch to H2).

**Comment by tomerd** [comment]
@swift-server-bot add to whitelist

**Comment by tomerd** [comment]
@swift-server-bot test this pease

**Comment by garnaat** [comment]
I assume you are maintaining some sort of version string for this package somewhere in the code.  If so, it might be a good idea to incorporate the version string into the User Agent string.  It would make it possible to track usage of the runtime across versions.

**Comment by tomerd** [comment]
good idea @garnaat https://github.com/swift-server/swift-aws-lambda-runtime/issues/108

**Comment by weissi** [comment]
@tomerd I filed https://bugs.swift.org/browse/SR-12926

**Pull request merged**